renku/data/pre-commit.sh

```diff
@@ -39,34 +39,6 @@ if [ ${#MODIFIED_FILES[@]} -ne 0 ] || [ ${#ADDED_FILES[@]} -ne 0 ]; then
 fi
 
 if [ ${#MODIFIED_FILES[@]} -ne 0 ] ; then
-  MODIFIED_OUTPUTS=$(renku show outputs "${MODIFIED_FILES[@]}")
```

Comment:
Should we create a followup story for this?

Reply:
I tested it and modifying outputs didn't seem to break anything, so I don't think we need a followup story